conn.go

```diff
@@ -138,6 +138,30 @@ func NewConnWith(conn net.Conn, config ConnConfig) *Conn {
 	return c
 }
 
+// GetController requests kafka for the current controller and returns its URL
+func (c *Conn) GetController() (controller string, err error) {
```

Comment:
How about just calling this `Controller` or may `ReadController` to match the pattern of other read operations?

Could the method return a `Broker` type? That would provide stronger type checking by the compiler than a string.

Let me know what you think!

Reply:
Thanks for your input. I would totally agree with naming it Controller. I can do that. I've thought about the return type and there is no Broker Type, yet. I can implement that for sure, but I didn't want to make any big proposals to the API. Another idea, which came to my mind was something like this:

```
// ControllerConn returns a connection to the controller of a cluster
func (c *Conn) ControllerConn() (controller *Conn, err error)
```

The downside of this is, that we would have to *reuse* the dial parameters. Maybe adding a dialer as parameter and if it's nill reusing the existing one? 

I'm open to any suggestions as long as I'm able to connect to the controller afterwards ;)

Reply:
Actually there is a `kafka.Broker` type https://godoc.org/github.com/segmentio/kafka-go#Broker

Obtaining a connection to the controller is an interesting idea, however it seems like it could be quite inefficient if all we need is the address. Maybe we can add another method on the `kafka.Dialer` type to do this, like `DialController`?